cmd/credential.go

```diff
@@ -30,7 +30,7 @@ const (
 	accessKeyMinLen = 5
 	accessKeyMaxLen = 20
 	secretKeyMinLen = 8
-	secretKeyMaxLen = 40
+	secretKeyMaxLen = 100
```

Comment:
problem with having one value for S3 and one for azure is we will have to maintain the logic/code for it.

i.e we can declare global variable

var secretKeyMaxLen

this would have to be initialized to 80 for S3 and 100 for azure. Simpler solution would be to keep it as one value.